src/authorizations/users.rs

```diff
@@ -1,6 +1,6 @@
 use crate::config::Config;
 
-pub fn is_authorized(id: &str, config: &Config) -> bool {
+pub fn is_authorized(id: &str, config: Config) -> bool {
```

Comment:
Hmm, you can probably still accept a reference here, passing `&config` to the function most of the time (like down below) or `&config.clone()` when you have lifetime issues.

Reply:
If you have `config: Arc<Config>`, you can get an `&Config` by dereferencing and borrowing, i.e. `&*config`.

Reply:
Done!